parsec/data_internal.h

```diff
@@ -80,6 +80,8 @@ struct parsec_data_copy_s {
     parsec_datatype_t         dtt;                   /**< the appropriate type for the network engine to send an element */
 };
 
+#define PARSEC_DATA_CREATE_ON_DEMAND ((parsec_data_copy_t*)(intptr_t)(-1))
```

Comment:
Why not NULL?

Reply:
Because NULL is already used to denote control flow